JuicingJuicer/solvd-automation
# Mobile HW

# Test Cases:
### 1. Test Battery Settings
1. Open Android settings - Android settings page is loaded
2. Click battery setting - Battery page is loaded
3. Turn on battery percent switch - Battery percent switch is turned on
4. Verify battery percent switch status - Battery percent switch status matches with expected results

### 2. Test Screen Flash Color
1. Open settings page - Settings page is loaded
2. Click search bar - Search bar page is loaded
3. Type in 'screen flash' - Screen flash search result appears
4. Click search result - Screen flash page is loaded
5. Click screen flash button - Color selection menu pops up
6. Select color based on given selection - Proper color is selected
7. Click done - Color selection menu is closed
8. Verify color selection - Color shown in summary section matches with expected results

### 3. Test Airplane Mode
1. Open settings page - Settings page is loaded
2. Click network button - Network page is loaded
3. Turn on airplane mode switch - Airplane mode switch is on
4. Read internet subheader - Internet subheader text is read
5. Read call subheader - Call subheader text is read
6. Read status of sim setting - Status of network page is read
7. Read status of airplane mode switch - Status of airplane mode switch is read
8. Verify all read items - All items read matches with expected results

### 4. Test West to East time zone
1. Open settings page - Settings page is loaded
2. Click system button - System page is loaded
3. Click date and time button - Date and time page is loaded
4. Turn off timezone auto switch - Time zone auto switch is turned off
5. Click time zone button within the date and time page - Time zone page is loaded
6. Click time zone button within the time zone page - Time zone selection menu is loaded
7. Select west time zone - West time zone is selected
8. Click back button - Date and time page is loaded
9. Read hours of west time zone - West time zone hours are read
10. Click time zone button within the date and time page - Time zone page is loaded
11. Click time zone button within the time zone page - Time zone selection menu is loaded
12. Select east time zone - East time zone is selected
13. Click back button - Date and time page is loaded
14. Read hours of east time zone - East time zone hours are read
15. Verify west and east time zone times - West and east time zone difference matches with expected results

### 5. Test Storage Size
1. Open settings page - Settings page is loaded
2. Read available storage size - Available storage size is read
3. Click storage button - Storage page loaded
4. Read storage size total - Storage size total is read
5. Read storage size used - Storage size used is read
6. Verify available storage size - Avaiable storage size matches with the difference of storage size total and storage size used
